pkg/overlay/cache.go

```diff
@@ -114,16 +107,35 @@ type UpdateRequest struct {
 	IsUp         bool
 }
 
+// NodeDossier is the complete info that the satellite tracks for a storage node
+type NodeDossier struct {
+	pb.Node
+	Type       pb.NodeType
+	Operator   pb.NodeOperator
+	Capacity   pb.NodeCapacity
+	Reputation NodeStats
+}
+
+// Online checks if a node is online based on the collected statistics.
+//
+// A node is considered online if the last attempt for contact was successful
+// and it was within the last hour.
+func (node *NodeDossier) Online() bool {
+	return time.Now().Sub(node.Reputation.LastContactSuccess) < 1*time.Hour &&
```

Comment:
could this 1*time.Hour be configurable somehow? hmm

Reply:
at very least might be nice to promote to a top level const

Reply:
what's the use case?

Reply:
to be honest, having it be configurable could potentially help with https://storjlabs.atlassian.net/browse/V3-1479, where we don't want the tests to have to wait for an hour before a down node is considered offline

Reply:
okay if this isn't fixed in this pr though

Reply:
I don't believe this would help the tests. Tests can call `UpdateUptime` to make nodes appear as offline after stopping them. This PR modifies the tests to do this.

Reply:
i'm still just anxious that i pulled the 1 hour number out of a hat and data science might discover that there's a better value for this and might like to try different things. i'm satisfied if it can have a top level const name and a comment

Reply:
It's now available as `overlay.OnlineWindow` const.